packages/discord.js/typings/index.d.ts

```diff
@@ -2625,6 +2625,11 @@ export class UserFlagsBitField extends BitField<UserFlagsString> {
   public static resolve(bit?: BitFieldResolvable<UserFlagsString, number>): number;
 }
 
+export interface WebhookCredentials {
+  id: string;
+  token: string;
+}
```

Comment:
Should this not be a `Snowflake`? And then, wouldn't it be the same as this?

https://github.com/discordjs/discord.js/blob/ffc439e58e020b98f7d36bc47186778c3ce803eb/packages/discord.js/typings/index.d.ts#L5462-L5465